src/kernel/orchestrator.py

```
@@ -0,0 +1,135 @@
+        for f in sorted(Path(target_dir).rglob("*")):
+            if not f.is_file():
+                continue
+            suffix = f.suffix.lstrip(".")
+            if suffix not in extensions:
+                continue
+            try:
+                ctx = self.analyze(str(f), library=library)
+                if ctx.analysis:
+                    reports.append(ctx.analysis)
+            except Exception:
+                pass
```

Ruff (0.15.18)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)